linera-storage-service/src/server.rs

```diff
@@ -45,10 +45,16 @@ enum ServiceStoreServerInternal {
     RocksDb(RocksDbStore),
 }
 
+#[derive(Default)]
+struct BigRead {
+    num_read: usize,
```

Comment:
When I read this I was confused on wether this was the expected total number of reads or what it was. So maybe `num_processed_chunks` or something like that would be a better name?

Reply:
Ok, done.